yaksh/views.py

```diff
@@ -771,12 +771,35 @@ def reject(request, course_id, user_id=None, was_enrolled=False):
         raise Http404('This course does not belong to you')
 
     if request.method == 'POST':
-        reject_ids = request.POST.getlist('check')
+        user_ids = request.POST.getlist('check')
+        if not user_ids:
+            message = "Please select atleast one User"
+            return my_render_to_response('yaksh/course_detail.html',
+                                        {'course': course, "message": message},
+                                        context_instance=ci)
+
+        if request.POST.get('send_mail') == 'send_mail':
```

Comment:
since in html form I cannot change action i.e. post request url hence I added send mail in reject function. I think to solve this issue is to use js for changing action url during form submit.
But is it ok?